src/automaton/levenshtein.rs

```diff
@@ -111,12 +111,10 @@ impl Levenshtein {
     #[inline]
     pub fn new(
         query: &str,
-        distance: u32,
+        distance: usize,
```

Comment:
Could you please say why? This seems trivial to me, and there is no way i'm going to put out a breaking change release for something like this given that I just released `0.4`.

Reply:
It makes sense in combination with the main change - `LevenshteinState` - to make all distance types consistent.